contracts/cw20-base/helpers.ts

```diff
@@ -202,7 +181,7 @@ interface CW20Contract {
   // codeId must come from a previous deploy
   // label is the public name of the contract in listing
   // if you set admin, you can run migrations on this contract (likely client.senderAddress)
-  instantiate: (codeId: number, initMsg: InitMsg, label: string, admin?: string) => Promise<CW20Instance>
+  instantiate: (codeId: number, initMsg: Record<string, unknown>, label: string, admin?: string) => Promise<CW20Instance>
```

Comment:
The TS compiler forces this on you?

Ideally it would auto downcast InitMsg to this more general `Record<string, unknown>`. And nice to have the clarity in the contract-specific API. If you have TS issues, leave it and I can look into it later.

Reply:
This is the error I get with v0.23:
```
../../../../../.npm/_npx/10245/lib/node_modules/@cosmjs/cli/[eval].ts:307:53 - error TS2345: Argument of type 'InitMsg' is not assignable to parameter of type 'Record<string, unknown>'.
  Index signature is missing in type 'InitMsg'.

307     const result = await client.instantiate(codeId, initMsg, label, { memo: `Init ${label}`, admin});
                                                        ~~~~~~~
```